src/constants/pools.ts

```diff
@@ -133,9 +133,11 @@ export const DOUBLE_ESTIMATE_POOLS_KOVAN = doubleEstimatePoolsKovan
 export const DEPOSIT_TXHASH: Record<string, string> = {
   '0x64dcbdeb83e39f152b7faf83e5e5673faca0d42a': '0xead0a9717cb13feff2ee58d78786b7b2eac68eafe61809d7bfecf67986222ba8', // xSGD:USDC mainnet
   '0xe15e50ff9d52bec41d53d3173f2ed40834d455f4': '0x6038346288968a18c766987b8a7811a2df17a25316edeb23800bb2125ba5902f', // TCAD:USDC mainnet
-  '0x2ed09e2961d72659e4002ba8c2badfedc7db19b7': '0xe1ba04f32987d607980ad89260942e16642ea74b23f1175a06dbc90ad0a30f28', // TGBP:USDC mainnet
+  // '0x2ed09e2961d72659e4002ba8c2badfedc7db19b7': '0xe1ba04f32987d607980ad89260942e16642ea74b23f1175a06dbc90ad0a30f28', // TGBP:USDC mainnet
   '0x20e1d8daf58358cf11be5616946e1df55f1ef8b0': '0x6dc4b1c61c00004c85ec953a0d00bd680bb42109977fb19bf4f4cfa7382910dc', // fxPHP:USDC mainnet
   '0x868084406449bda10a5bd556fb33cef5086b0797': '0xa605e6a3bf8c64b1a9f3e835dda1e61a77027789a6bde64dab43f91657d29081', // UST:USDC mainnet
-  '0x8123c64d6607412c7ac9e880f12245ef22558b14': '0x1baa65bca924e1a189ae68c0a57768d49f17e661de231566a2236383663f3fc7', // xSGD:USDC matic
-  '0x6156f030b877344470bac075a708d7e27602cc17': '0xafb8144ceea36323878efa628cbfcd41a01e91d931a61edb771d912fe2173920' // tagPHP:USDC matic
+  '0x8123c64d6607412c7ac9e880f12245ef22558b14': '0x1baa65bca924e1a189ae68c0a57768d49f17e661de231566a2236383663f3fc7' // xSGD:USDC matic
+  // '0x6156f030b877344470bac075a708d7e27602cc17': '', // tagPHP:USDC matic
```

Comment:
@geonellov can we enable

- tagPHP:USDC matic
- fxPHP:USDC arbitrum?

Reply:
@geonellov this is marked as resolved but I don't see the base APR for the pools above